ursula_cli/shell.py

```diff
@@ -527,6 +542,14 @@ def parse_args():
         '--heat-stack-name', default=None,
         help='Name of the heat stack when heat provisioner is used',
     )
+    parser.add_argument(
+        '--heat-stack-update', default=False, action='store_true',
+        help='Update the heat stack',
+    )
+    parser.add_argument(
+        '--heat-parameters', default=None,
+        help='Semicolon separated list of KEY=VALUE parameters for heat',
```

Comment:
Looking at the signature for format_parameters, it looks like you can omit the semicolon parsing if you want.  The reason I'm saying that we might want to simplify heat's weird choice here because the semicolon is a shell reserved symbol, so we'll need to escape it or quote it.

`heat -P foo=bar;baz=qux` would really need to be `heat -P foo=bar;baz=qux` or `heat -P foo=bar\;baz=qux` or something.